src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you can change this line to `font-family: Roboto, Arial, sans-serif;` to ensure that there's a default font if Roboto is not available.